### PR TITLE
fix(www): move doc-only components to unpublished lib 

### PR DESCRIPTION
**Pull request by eckdanny-osi**
### Jira Story

- [unpublished lib for docs components CODE-103](https://osi-cwds.atlassian.net/browse/CODE-103)

### CODE REVIEW

File: apps/www/package.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
   "license": "AGPL-3.0",
   "private": true,
   "dependencies": {
-    "@cwds/components": "^0.6.1",
-    "@cwds/core": "^0.6.1",
+    "@cwds/components": "^0.6.0",
```
Comment (eckdanny-osi) [inline apps/www/package.json]: this isnt right

File: apps/www/src/App.js [hunk 2]
```diff
@@ -29,16 +29,15 @@ class App extends Component {
...
-                <Route path="/core-style" component={CoreStyles} />
-                <Route path="/labs" component={LabsPage} />
-                <Route path="/components" component={Components} />
-                <Route path="/status" component={Status} />
-                <JumpToTop />
-              </Fragment>
-            </ScrollToTop>
-          </Router>
-        </CaresProvider>
+                <Route exact path="/" component={Home} />{' '}
```
Comment (eckdanny-osi) [inline apps/www/src/App.js]: ¿auto-format?

File: libs/docs/README.md.ORIG [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# `@cwds/docs`
```
Comment (eckdanny-osi) [inline libs/docs/README.md.ORIG]: can delete this file 🗑

File: apps/www/src/App.js (hunk 2 above)
Comment (NagaPradeepGajula-osi) [reply apps/www/src/App.js]: yup autoformat

File: apps/www/package.json (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply apps/www/package.json]: will fix it. Probably git master merge went wrong

File: libs/docs/README.md.ORIG (hunk 3 above)
Comment (NagaPradeepGajula-osi) [reply libs/docs/README.md.ORIG]: sure will delete

File: apps/www/package.json (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply apps/www/package.json]: done

File: libs/docs/README.md.ORIG (hunk 3 above)
Comment (NagaPradeepGajula-osi) [reply libs/docs/README.md.ORIG]: done

File: apps/www/src/App.js (hunk 2 above)
Comment (eckdanny-osi) [reply apps/www/src/App.js]: This actually _breaks_ the `www` app:

<details>
<summary><code>A &lt;Router&gt; may have only one child element</code></summary>

```
Uncaught Error: A <Router> may have only one child element
    at invariant (browser.js:34)
    at Router.componentWillMount (Router.js:97)
    at callComponentWillMount (react-dom.development.js:12286)
    at mountClassInstance (react-dom.development.js:12377)
    at updateClassComponent (react-dom.development.js:14059)
    at beginWork (react-dom.development.js:14687)
    at performUnitOfWork (react-dom.development.js:17242)
    at workLoop (react-dom.development.js:17281)
    at HTMLUnknownElement.callCallback (react-dom.development.js:144)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:193)
    at invokeGuardedCallback (react-dom.development.js:243)
    at replayUnitOfWork (react-dom.development.js:16536)
    at renderRoot (react-dom.development.js:17374)
    at performWorkOnRoot (react-dom.development.js:18252)
    at performWork (react-dom.development.js:18159)
    at performSyncWork (react-dom.development.js:18132)
    at requestWork (react-dom.development.js:18009)
    at scheduleWork (react-dom.development.js:17802)
    at scheduleRootUpdate (react-dom.development.js:18523)
    at updateContainerAtExpirationTime (react-dom.development.js:18549)
    at updateContainer (react-dom.development.js:18580)
    at ReactRoot.push../node_modules/react-dom/cjs/react-dom.development.js.ReactRoot.render (react-dom.development.js:18862)
    at react-dom.development.js:19016
    at unbatchedUpdates (react-dom.development.js:18397)
    at legacyRenderSubtreeIntoContainer (react-dom.development.js:19012)
    at Object.render (react-dom.development.js:19076)
    at Module../src/index.js (index.js:8)
    at __webpack_require__ (bootstrap:782)
    at fn (bootstrap:150)
    at Object.0 (serviceWorker.js:129)
    at __webpack_require__ (bootstrap:782)
    at checkDeferredModules (bootstrap:45)
    at Array.webpackJsonpCallback [as push] (bootstrap:32)
    at main.chunk.js:1
```
</details>

&nbsp;
The space literal `{' '}` is a React node and _cannot_ be both sibling to `<ScrollToTop />` _and_ a direct child of `<Router>`

I'm unclear why this is getting auto-formatted in the 1st place... `yarn lint` does not complain about it nor does prettier do anything with the file on save. Do you have some other editor config at play?

Comment (NagaPradeepGajula-osi) [reply apps/www/src/App.js]: Done

**Pull request merged**